src/utilities/__tests__/extendSchema-test.js

```diff
@@ -314,24 +372,54 @@ describe('extendSchema', () => {
       extend type Foo {
         deprecatedField: String @deprecated(reason: "not used anymore")
       }
+
+      extend enum SomeEnum {
+        DEPRECATED @deprecated(reason: "do not use")
+      }
     `);
     const deprecatedFieldDef = extendedSchema.getType('Foo').getFields()
       .deprecatedField;
     expect(deprecatedFieldDef.isDeprecated).to.equal(true);
     expect(deprecatedFieldDef.deprecationReason).to.equal('not used anymore');
+
+    const deprecatedEnumDef = extendedSchema
+      .getType('SomeEnum')
+      .getValue('DEPRECATED');
+    expect(deprecatedEnumDef.isDeprecated).to.equal(true);
+    expect(deprecatedEnumDef.deprecationReason).to.equal('do not use');
   });
 
   it('extends objects by adding new unused types', () => {
     const extendedSchema = extendTestSchema(`
       type Unused {
         someField: String
       }
+
+      enum UnusedEnum {
+        SOME
+      }
+
+      input UnusedInput {
+        someInput: String
+      }
+
+      union UnusedUnion = Unused
```

Comment:
I think 'extends schema by adding new unused types' would be better name for this test.

> union UnusedUnion = Unused

`Unused` became used 😄 It's better to create separate `DummyUnionMember` type.